docs/examples/audio/spectrogram.ipynb

```diff
@@ -0,0 +1,272 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "librosa.display.specshow(spectrogram_librosa_db, sr=sr, y_axis='log', x_axis='time', hop_length=hop_length)\n",
+    "plt.title('Librosa power spectrogram')\n",
+    "plt.colorbar(format='%+2.0f dB')\n",
+    "plt.tight_layout()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Calculating the spectrogram using DALI\n",
+    "\n",
+    "To demonstrate DALI's Spectrogram operator we will define a DALI pipeline, whose input will be provided externally with the help of ExternalSource operator. For demonstration purposes, we can just feed the same input in every iteration, as we will be only calculating one spectrogram."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from nvidia.dali.pipeline import Pipeline\n",
+    "import nvidia.dali.ops as ops\n",
+    "import nvidia.dali.types as types\n",
+    "import nvidia.dali as dali\n",
+    "\n",
+    "class SpectrogramPipeline(Pipeline):\n",
+    "    def __init__(self, device, batch_size, nfft, window_length, window_step, num_threads=1, device_id=0):\n",
+    "        super(SpectrogramPipeline, self).__init__(batch_size, num_threads, device_id)\n",
+    "        self.device = device\n",
+    "        \n",
+    "        self.batch_data = []\n",
+    "        for _ in range(batch_size):\n",
+    "            self.batch_data.append(np.array(y, dtype=np.float32))\n",
```

Comment:
It is hard to tell that this `y` comes from `librosa.load(librosa.util.example_audio_file())`.
I would again load this just to make it more verbose.

Reply:
will do